raptor/tests/raptor-chrome-tp7.ini

```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# raptor tp7 chrome
+
+[DEFAULT]
+apps = chrome
+type =  pageload
+playback_tool = mitmproxy
+page_cycles = 25
+
+[raptor-chrome-tp7]
+test_url = http://localhost:8081/heroes
+measure = fcp,hero
```

Comment:
Not really sure the best way to format multiple items on one INI file key.... what do you think? Comma separated doesn't look that great visually.

Reply:
I prefer space delimiting with one item per line (like in the link from the other comment).